3-projects/modules/folder_environments/README.md

```diff
@@ -0,0 +1,16 @@
+<!-- BEGINNING OF PRE-COMMIT-TERRAFORM DOCS HOOK -->
```

Comment:
Modules should go in a root-level modules directory.

Reply:
I'll add an issue for this, as we should refactor the network code in the same style https://github.com/terraform-google-modules/terraform-example-foundation/issues/11

Reply:
Actually on second thoughts, we are currently following this convention. Currently each of the subfolders has its own repo & pipeline, modules are at the top level within each of those repos. At the moment the codebase is being delivered through one repo for convenience, but they are designed to be iterated on independently in practice through seperate git repos.